test/e2e/aws_eip_for_nlb_test.go

```diff
@@ -0,0 +1,492 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/api/features"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	operatorclient "github.com/openshift/cluster-ingress-operator/pkg/operator/client"
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	awsLBEIPAllocationAnnotation = "service.beta.kubernetes.io/aws-load-balancer-eip-allocations"
+)
+
+func CreateAWSEIPs() ([]string, error) {
+	kubeClient, err := createClient()
+	if err != nil {
+		return nil, fmt.Errorf("test skipped: %v", err)
```

Comment:
all these say "test skipped", I think that's a mistake

Reply:
do you mean the message does not sound correct ?

Reply:
Right. I don't think this is skipping the test.

Reply:
yes added meaningful statements.